x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/TokenServiceTests.java

```diff
@@ -253,7 +253,7 @@ private void rotateKeys(TokenService tokenService) {
 
     public void testKeyExchange() throws Exception {
         TokenService tokenService = new TokenService(tokenServiceEnabledSettings, systemUTC(), client, securityIndex, clusterService);
-        int numRotations = 0;randomIntBetween(1, 5);
+        int numRotations = randomIntBetween(1, 5);
```

Comment:
I also opened a separate PR for this change: https://github.com/elastic/elasticsearch/pull/33681